filecoin-proofs/src/bin/circuitinfo.rs

```diff
@@ -0,0 +1,257 @@
+use dialoguer::{theme::ColorfulTheme, MultiSelect};
+use humansize::{file_size_opts, FileSize};
+use log::{info, warn};
+use structopt::StructOpt;
+
+use bellperson::util_cs::bench_cs::BenchCS;
+use bellperson::Circuit;
+use filecoin_proofs::constants::*;
+use filecoin_proofs::parameters::{
+    public_params, window_post_public_params, winning_post_public_params,
+};
+use filecoin_proofs::types::*;
+use filecoin_proofs::with_shape;
+use filecoin_proofs::PoStType;
+use paired::bls12_381::Bls12;
+use storage_proofs::compound_proof::CompoundProof;
+use storage_proofs::porep::stacked::{StackedCompound, StackedDrg};
+use storage_proofs::post::fallback::{FallbackPoSt, FallbackPoStCircuit, FallbackPoStCompound};
+
+const PUBLISHED_SECTOR_SIZES: [u64; 10] = [
```

Comment:
I wonder if we should share this from https://github.com/filecoin-project/rust-fil-proofs/blob/master/filecoin-proofs/src/bin/paramcache.rs#L20 (or refactor both to share from a common place).

Reply:
Probably.

Reply:
Moved to `constants.rs`.